internal/cmd/local/command_kafka.go

```diff
@@ -0,0 +1,30 @@
+package local
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+const imageName = "523370736235.dkr.ecr.us-west-2.amazonaws.com/confluentinc/kafka-local:latest"
```

Comment:
Will there be a public image available soon?

Reply:
expected end of q1